File: go/raindrops/raindrops.go
package raindrops

import (
	"strconv"
)

// Convert converts an into into a raindrops string.
func Convert(num int) string {
	output := ""

	if num%3 == 0 {
		output += "Pling"
	}

	if num%5 == 0 {
		output += "Plang"
	}

	if num%7 == 0 {
		output += "Plong"
	}

	if output == "" {
		return strconv.Itoa(num)
	}

	return output
}

File: lua/raindrops/raindrops.lua
return function(n)
	local raindrop = ''

	if n % 3 == 0 then
		raindrop = raindrop..'Pling'
	end

	if n % 5 == 0 then
		raindrop = raindrop..'Plang'
	end

	if n % 7 == 0 then
		raindrop = raindrop..'Plong'
	end

	if raindrop == '' then
		return tostring(n)
	end

	return raindrop
end
